javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithExpression.java

```diff
@@ -0,0 +1,10 @@
+package com.github.javaparser.ast.nodeTypes;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.expr.Expression;
+
+public interface NodeWithExpression<N extends Node> {
```

Comment:
What are those nodes with Expressions? Are Expressions wrapping other expressions? Any sort of node containing one single expression?